injectors/main.js

```diff
@@ -87,3 +89,12 @@ exports.uninject = async ({ getAppDir }, platform) => {
   await rmdirRf(appDir);
   return true;
 };
+
+exports.checkPlatform = (platform) => {
+  return this.VALID_PLATFORMS.includes(platform);
+};
+
+exports.checkInstalled = async ({ getAppDir }, platform) => {
+  const appDir = await getAppDir(platform);
+  return existsSync(join(appDir, '..'));
+};
```

Comment:
Let's move these back to index and just put them inline. Move the const above too. I don't see any need for them to be in this file.